docs/developer.md

```diff
@@ -36,6 +39,17 @@ You should set up the correct `REGION` env variable
 export REGION=us-west-2
 make e2etest
 ```
+
+Pass `FOCUS` environment variable to run some specific test cases based on filter condition:
+```bash
+export REGION=us-west-2
+export FOCUS="HTTPRoute should support multiple path matches" 
```

Comment:
Please add a note on how to find out the this `FOCUS` string to a particular test.  thanks

Reply:
Ok, Added more description in the next revision